src/molecules/formfields/CheckBoxField/index.js

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+
+import toggleClass from 'utils/ReactUtils/toggleClass';
+
 import styles from '../shared/formfields.module.scss';
 
 function CheckboxList( props ) {
   const {
     label,
     focused,
     data,
-    handleClick
+    handleClick,
+    meta,
   } = props;
 
+  const focusedClass = toggleClass(focused || (meta && meta.active), styles['focused']);
```

Comment:
I think you can probably remove the `focused` prop entirely because we can still programmatically set the focused state through the `meta` object (if we're not using redux-form)